zoom/client.go

```diff
@@ -18,8 +18,8 @@ import (
 )
 
 const (
-	authURL = "https://zoom.us/oauth/token"
-	baseURL = "https://api.zoom.us/v2"
+	authURL     = "https://zoom.us/oauth/token"
```

Comment:
```suggestion
	zoomAuthURL     = "https://zoom.us/oauth/token"
```
I know baseURL was changed to prevent shadowing but this would be nice for consistency